extensions/amp-cache-url/0.1/amp-cache-url.js

```diff
@@ -14,11 +14,24 @@
  * limitations under the License.
  */
 
+import * as ampToolboxCacheUrl from '@ampproject/toolbox-cache-url';
+import {urls} from '../../../src/config';
+
 export class AmpCacheUrlService {
   /**
    * Create cache url service
    */
   constructor() {}
+
+  /**
+   *
+   * @param {string} url
+   * @param {string=} cache
+   * @return {!Promise<string>}
+   */
+  createCacheUrl(url, cache = urls.cdn.replace('https://', '')) {
```

Comment:
Should the provided `url` param or cache param have/not have `https://`?

Reply:
It should not have the `https://`, the value we want is `cdn.ampproject.org` but it cannot be hardcoded as the PR checks will complain.

Reply:
Suggestion: unless it's validated somewhere else in this file, note that expectation in the JSDoc comment for the param and/or apply the replacement to `url` and `cache`

Reply:
Will add a comment and do the replacement inside the function so the param doesn't have the `https?://`